index.bs

```diff
@@ -730,6 +822,15 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} of the {{VRDisplay}} this {{Gamepad}} is associated with. A {{Gamepad}} is considered to be associated with a {{VRDisplay}} if it reports a pose that is in the same space as the {{VRDisplay}} pose. If the {{Gamepad}} is not associated with a {{VRDisplay}} should return 0.
 
+<dfn attribute for="Gamepad">hasPosition</dfn>
```

Comment:
Here's a thought: Seems like the concepts of having a position/orientation/etc are something that is actually tied into the FrameOfReference? The attached coordinate system has no position, while the others do, etc. Would it be reasonable to move these capability bits onto the FrameOfReference, and then make it so that the devices capabilities are inferred based on the FramesOfReference that it supports? that would eliminate the need to awkwardly patch these values into the Gamepad, at least. Or maybe I'm overthinking it. :)